llama-index-core/llama_index/core/retrievers/fusion_retriever.py

```diff
@@ -88,7 +89,12 @@ def _get_queries(self, original_query: str) -> List[QueryBundle]:
         response = self._llm.complete(prompt_str)
 
         # assume LLM proper put each query on a newline
-        queries = response.text.split("\n")
+        raw_queries = response.text.split("\n")
+        # Use strip and regex to split the query based on numbering points to avoid empty query(s).
+        queries = []
+        for query in raw_queries:
+            query = [q.strip() for q in re.split(r"\d+\.", query) if q.strip()]
```

Comment:
We don't specifically ask for number queries, just one on each line

I would just filter out any blank strings after doing the split instead?

Reply:
You are right. I have update the request to only filter out blank query.